prefect_census/client.py

```diff
@@ -0,0 +1,94 @@
+"""Module containing client for interacting with the Census API"""
+from httpx import AsyncClient, Response
+
+
+class CensusClient:
+    """
+    Client for interacting with the Census API.
+
+    Args:
+        api_key: API key to authenticate with the Census API.
+    """
+
+    def __init__(self, api_key: str):
+        self._closed = False
+        self._started = False
+
+        self.client = AsyncClient(
+            base_url="https://app.getcensus.com",
+            headers={"Authorization": f"Bearer {api_key}"},
+        )
+
+    async def call_endpoint(
+        self,
+        http_method: str,
+        path: str,
+    ) -> Response:
+        """
+        Call an endpoint in the Census API.
+
+        Args:
+            http_method: HTTP method to call on the endpoint.
+            path: The partial path for request (e.g. //api/v1/syncs/42). Will be
+            appended onto the base URL as determined by the client configuration.
```

Comment:
New lines for a description should be indented
```suggestion
                appended onto the base URL as determined by the client configuration.
```